client/services/FeedbackGeneratorService.js

```diff
@@ -54,11 +54,12 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     return {
-      getFeedback: function(prompt, codeEvalResult) {
+      getFeedback: function(
+          prompt, codeEvalResult, rawCodeLineIndexes) {
```

Comment:
Ah, I see. Yep, this didn't trigger in the tests because that code path wasn't executed, oops. I'll make a note to fix it in a follow-up. Thanks for catching!